python/ray/autoscaler/_private/kuberay/utils.py

```diff
@@ -103,7 +104,7 @@ def tpu_node_selectors_to_type(topology: str, accelerator: str) -> Optional[str]
         chip_dimensions = [int(chip_count) for chip_count in topology.split("x")]
         num_chips = reduce(lambda x, y: x * y, chip_dimensions)
         default_num_cores_per_chip = 2
```

Comment:
Consider inverting this since more generations default to 1 core per chip now
```
default_num_cores_per_chip = 1
if generation == "v4":
  default_num_cores_per_chip = 2
```

Reply:
9ad2663